ports/raspberrypi/boards/adafruit_feather_rp2040/pins.c

```diff
@@ -27,6 +27,7 @@ STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_D13), MP_ROM_PTR(&pin_GPIO13) },
 
     { MP_ROM_QSTR(MP_QSTR_NEOPIXEL), MP_ROM_PTR(&pin_GPIO16) },
+    { MP_ROM_QSTR(MP_QSTR_LED), MP_ROM_PTR(&pin_GPIO13) },
```

Comment:
Could you put this adjacent to the D13 def, below line 27?

Reply:
@dhalbert done!